City-of-Helsinki/drupal-hdbt
# UHF-10619: Double space in hakuvahti keyword will make the hakuvahti useless.

# [UHF-10619](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10619)

## What was done
Prevent adding double space in job search keyword.

## How to install

* Make sure your instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-10619`
* Run `make drush-cr`

## How to test
- Go to job search and try to add double space to the keyword field.
It should not be possible to add 2 spaces to search keyword.




[UHF-10619]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10619?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
